app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,15 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str, favourite_dish: str,
+                 hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf warrior {self.nickname}. {self.nickname} "
+                + f"has a hummer of the {self._hummer_level} level")
+
+    def get_rating(self) -> int:
+        return self._hummer_level + 4
```

Comment:
The `get_rating` method returns the `hummer_level` plus 4, which is appropriate for this class. Ensure that this method aligns with the expected behavior in the context where it's used.